extension/entitystore/ec2Info.go

```diff
@@ -48,8 +49,11 @@ func (ei *EC2Info) initEc2Info() {
 	if err := ei.setInstanceIDAccountID(); err != nil {
 		return
 	}
-	if err := ei.setAutoScalingGroup(); err != nil {
-		return
+	// Instance metadata tags is not usable for EKS nodes
+	// https://github.com/kubernetes/cloud-provider-aws/issues/762
+	if ei.kubernetesMode == "" {
```

Comment:
we want to keep the retryer on native k8s on ec2 right?

Reply:
Fixed in new commit